tests/sentry/incidents/endpoints/test_project_alert_rule_index.py

```diff
@@ -632,7 +632,7 @@ def test_offset_pagination(self):
         self.three_alert_rule = self.create_alert_rule(projects=self.projects)
 
         with self.feature(["organizations:incidents", "organizations:performance-view"]):
-            request_data = {"per_page": "2"}
+            request_data = {"per_page": "2", "sort": ["date_added", "id"]}
```

Comment:
Does that work here? I'm not sure if the paginator respects us passing the sort https://github.com/getsentry/sentry/blob/d745edbd591063f2c3241cd1960c361834058823/src/sentry/incidents/endpoints/project_alert_rule_index.py#L29-L54.

Tbh I'm not sure if we can even change the sort in the endpoint, since I'm not sure if that combined paginator actually supports multiple cols, but you could try it.

Reply:
This endpoint is also barely used, I have a feeling that one customer is just scraping it intermittently: https://app.datadoghq.com/metric/explorer?start=1663792562866&end=1664397362866&paused=false#N4IgNghgng9grgFxALlADxQRgDQil3AdwEsATBACwJAoFNiBzCpZTAX11NoDNiA7YgmIw+KUAigAHWihBCAtrQDOtAE7FlIXKtoBHOMoRKUAbVA6lkkSoD63GKvkQWc4opXrNufWo3HkZiB8EIqyPqpQmFogpM4QNkrwqgDGMsggigjqycbeBhGyEABuDMgqfFlQAHRFGoRVFlZ8KsD8SggQfKlltBURVfzJZL1GVb2kVvyjkqowAFa0yQg2EGBqy6pwazb8XGhVM-OLCMkw8gBG-LSkm2u7tGjjkxVsVRBKNqdwFQAUAJQgNgAXVw9kcW3eplAYPkELC+UigJBIHaUDWYhAklWtAQCDSMRgDE+kCUSmIyWiYCuNgk0lkiSppEp1JI5Co6T4DicYEBnGIlkgUBpUnx5wgqmMwLYbCAA

We could probably kill most of the code and just share code with https://github.com/getsentry/sentry/blob/d745edbd591063f2c3241cd1960c361834058823/src/sentry/incidents/endpoints/organization_alert_rule_index.py#L32. Probably a bit much for this pr though

Reply:
Right .. I confused this endpoint with `OrganizationCombinedRuleIndexEndpoint` which looks to support multiple sort values. Will take a deeper look later.